flask_project/database_utils.py

```diff
@@ -0,0 +1,53 @@
+"""
+database_utils.py
+
+This module stands for database management functions.
+Adding new records, retrieving all records etc.
+"""
+from datetime import datetime
+from .models import db, URL, IP_address, Domains
+
+
+def add_new_records(ipqs_data: dict, url: str):
```

Comment:
I think better to have separate functions for the objects creation(eg. function for IP_address instance creation from ipqs_data). It will make the code more readable.